src/Flux/neural_de.jl

```diff
@@ -21,12 +21,12 @@ kwargs key word arguments passed to ODESolve; accepts an additional key
     passes a separate callback to the adjoint solver.
 
 """
-function neural_ode(model,x,tspan,
-                    args...;kwargs...)
+function neural_ode(model,x,tspan,args...;kwargs...)
   p = destructure(model)
-  dudt_(u::TrackedArray,p,t) = restructure(model,p)(u)
-  dudt_(u::AbstractArray,p,t) = Tracker.data(restructure(model,p)(u))
-  prob = ODEProblem(dudt_,x,tspan,p)
+  _model = restructure(model,p)
+  dudt_(u::TrackedArray,p,t) = _model(u)
```

Comment:
@ChrisRackauckas Zygote compatibility progress! Awesome! these lines will go away also, right? No need to dispatch separately on Tracked Arrays now.

Reply:
For now we need to keep Tracker compatibility for the adjoint pass.